src/main/java/com/edmunds/rest/databricks/service/JobServiceImpl.java

```diff
@@ -206,6 +206,8 @@ public RunsDTO listRuns(Long jobId, Boolean activeOnly, Integer offset, Integer
     }
     if (activeOnly != null) {
       data.put("active_only", activeOnly);
+    } else {
+      data.put("completed_only", true);
```

Comment:
Why is this filter necessary?

Reply:
hmm it was necessary for billing calculations. But I can see that our function as it is now doesn't imply that completed_only would be set if activeOnly is false. Will remove